app/controllers/legacy_controller.rb

```diff
@@ -19,13 +19,13 @@ def people
     redirect_to URI.parse('/profile/' + params[:id]).path, status: 301
   end
 
-  def place
-    redirect_to URI.parse('/wiki/' + params[:id]).path, status: 301
-  end
+  # def place
+  #   redirect_to URI.parse('/wiki/' + params[:id]).path, status: 301
+  # end
 
-  def tool
-    redirect_to URI.parse('/wiki/' + params[:id]).path, status: 301
-  end
+  # def tool
+  #   redirect_to URI.parse('/wiki/' + params[:id]).path, status: 301
+  # end
 
   def openid
     user = User.find params[:id]
```

Comment:
This file is not included in the list of files @cesswairimu mentioned in https://github.com/publiclab/plots2/issues/714#issuecomment-832061661

Reply:
> This file is not included in the list of files @cesswairimu mentioned in [#714 (comment)](https://github.com/publiclab/plots2/issues/714#issuecomment-832061661)

@TildaDares, I see that commented part of the code is not used anywhere as we have removed references to 'Place' and 'Tool'. Please correct me if wrong.

Reply:
@KSVSC Yeah you're right but place is used in https://github.com/publiclab/plots2/blob/main/app/views/locations/_form.html.erb#L53. I think @cesswairimu will know how to handle this.

Reply:
@KSVSC I think you can add this to the list of files mentioned by @cesswairimu in her comment https://github.com/publiclab/plots2/issues/714#issuecomment-832061661 ... I do think that the reference to `tool `and `place` here should be removed but just in case anything else breaks we want to have an isolated case with few changes.